components/sections/Buy.js

```diff
@@ -0,0 +1,130 @@
+import Link from "next/link";
+
+
+export default function Wallets() {
+
+    const title = "How to get Koin";
+    const subtle = "You can receive it from your peers, or buy it from exchanges ";
```

Comment:
Might look cleaner without the 'subtle' line.. just "How to get $KOIN", or even better, "Where to buy $KOIN". Obviously up to you guys though